pkgs/development/tools/kube-shell/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib, pythonPackages, fetchFromGitHub }:
+
+let
+  inherit (pythonPackages) python;
+in
+pythonPackages.buildPythonApplication rec {
+  name = "kube-shell-${version}";
+  version = "0.0.21";
+
+  src = fetchFromGitHub {
+    sha256 = "0966szv0n6mpsxfphv8z4h1h1rikhx4k8z3v5p9b2vmmw8048rn5";
+    rev = "v${version}";
+    repo = "kube-shell";
+    owner = "cloudnativelabs";
+  };
+
+  buildInputs = with pythonPackages; [ pexpect ];
+
+  buildPhase = ''
```

Comment:
You should be able to use the builtin buildPhase provided by buildPythonApplication